dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -112,15 +111,15 @@ public boolean processDailyFiles() {
 
     // Component Methods
 
-    // Should this be kept? This is more for readability/formatting. Feels wrong.
-    private List<ConcorContribEntry> getContributionList(int startingId) {
+    // TODO: Should this be kept? This is more for readability/formatting. Feels wrong.
```

Comment:
This is from David I think and perhaps safe to remove if most of us agree that it is superfluous.

Reply:
Yep. All good.